src/i18n/en/nav.ts

```diff
@@ -48,6 +48,7 @@ export default [
 		key: 'guides/content-collections',
 	},
 	{ text: 'Connect a CMS', slug: 'guides/cms', key: 'guides/cms' },
+	{ text: 'Connect Hosted Media or DAM', slug: 'guides/media', key: 'guides/media' },
```

Comment:
I was thinking about this going under Assets actually!

Reply:
yeah makes sense to me! i misunderstood and thought we were relating it to the content section

the only thing ot think about in the future could be  video? but no reason that couldn't just fall in somewhere there still under Assets similar to how Image itself is set up 🤷 

pushign that change